swagger.yaml

```diff
@@ -7223,3 +7289,13 @@ definitions:
         type: string
         format: date-time
         description: Expiration time for the URL token.
+  
+  installer-ui-settings:
```

Comment:
If this is already an unstructured json blob, I'm not sure there's any reason to create a separate object with a single field. Can we not just take in and return the string directly?

@avishayt any thoughts on this?

Reply:
Regarding the overly complex request object for the JSON blob, I agree, however I think we have ran into this issue before where anything submitted as part of an in: body clause in swagger needs to be defined as a schema

Reply:
and adding this to query params seems unwise

Reply:
Looks like you can just define it as a string.
We do it for install config overrides here:
https://github.com/openshift/assisted-service/blob/7d2554116546ab0ce0d26c8d9de8be0f77e3d6d7/swagger.yaml#L2288-L2293